qiita_db/test/test_meta_util.py

```diff
@@ -46,13 +46,13 @@ def test_get_accessible_filepath_ids(self):
         obs = qdb.meta_util.get_accessible_filepath_ids(
             qdb.user.User('shared@foo.bar'))
         self.assertItemsEqual(obs, {
-            1, 2, 3, 4, 5, 9, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21})
+            1, 2, 3, 4, 5, 9, 12, 16, 17, 18, 19, 20, 21})
```

Comment:
Just to confirm, what you are saying is that those filepaths_ids where pushed but are not actually valid, right? If that's the case, are you creating new ones to test that the validate works or what's the plan? 

My concern is that: we don't add some kind of permission checking for the analysis.